lua/ulx/modules/sh/cfc_constraint_checker.lua

```diff
@@ -0,0 +1,129 @@
+CFCUlxCommands.constrainChecker = CFCUlxCommands.constrainChecker or {}
+local cmd = CFCUlxCommands.constrainChecker
+local CATEGORY_NAME = "Utility"
+local IsValid = IsValid
+
+-- Returns the owner of the entity if it is checkable, otherwise returns false
+local function getOwnerIfCheckable( ent )
```

Comment:
I don't see the need for `IfCheckable`, that's not really what the function is doing. All it does is get the owner.

Reply:
yeah, even getOwner() would be more logical